packages/javascript/bh-shared-ui/src/components/FileIngest/FileIngest.tsx

```diff
@@ -58,37 +58,33 @@ const FileIngest = () => {
 
     return (
         <>
-            <ContentPage title='Manual File Ingest' data-testid='manual-file-ingest'>
-                <Typography variant='body1'>
-                    <p>
-                        Upload data from SharpHound or AzureHound offline collectors. Check out our {fileIngestLink}{' '}
-                        documentation for more information.
-                    </p>
+            <ContentPage title='File Ingest' data-testid='manual-file-ingest'>
+                <Typography variant='body2'>
+                    Upload data from SharpHound or AzureHound offline collectors. Check out our {fileIngestLink}{' '}
+                    documentation for more information.
                 </Typography>
-                <Box display='flex' justifyContent='flex-end' alignItems='center' minHeight='24px' mb={2}>
-                    <Button
-                        color='primary'
-                        variant='contained'
-                        disableElevation
-                        onClick={() => toggleFileUploadDialog()}
-                        data-testid='file-ingest_button-upload-files'>
-                        Upload File(s)
-                    </Button>
-                </Box>
             </ContentPage>
 
-            <ContentPage title='Finished Ingest Log' data-testid='finished-ingest-log'>
-                <FinishedIngestLog
-                    ingestJobs={listFileIngestJobsData?.data || []}
-                    paginationProps={{
-                        page,
-                        rowsPerPage,
-                        count: totalCount,
-                        onPageChange: handlePageChange,
-                        onRowsPerPageChange: handleRowsPerPageChange,
-                    }}
-                />
-            </ContentPage>
+            <Box display='flex' justifyContent='flex-end' alignItems='center' minHeight='24px' mb={2}>
```

Comment:
Would you please update the `mb` prop on this box to instead be an `my` prop? If one is viewing this at a smaller resolution, the help text is too close to the button.

only margin on bottom at smaller resolution:

![Screenshot 2024-06-26 at 4 22 49 PM](https://github.com/SpecterOps/BloodHound/assets/16910931/47b9e5d6-0271-4429-87e4-dae0bac99d2f)


after updating to `my`:

![Screenshot 2024-06-26 at 4 26 54 PM](https://github.com/SpecterOps/BloodHound/assets/16910931/b9428a58-4489-44f8-b486-d8dbbdf7498d)